mne/viz/_figure.py

```diff
@@ -447,6 +447,15 @@ def __init__(self, inst, figsize, ica=None, xlabel='Time (s)', **kwargs):
             # hide some ticks
             ax_main.tick_params(axis='x', which='major', bottom=False)
             ax_hscroll.tick_params(axis='x', which='both', bottom=False)
+        else:
+            # Add Timestamp X-Ticks if activated
+            if self.mne.show_real_time:
```

Comment:
Why not combine this into an `elif` branch?

Reply:
I'll do it. Do you agree with the decision to exclude the epochs-plot? I thought, the application of this feature would be most helpful with continous data.

Reply:
Yes, I agree it's most useful for continuous plots.